lib/pubid/iso/parser.rb

```diff
@@ -109,7 +109,10 @@ class Parser < Parslet::Parser
         (type | stage.as(:stage))).maybe >>
         # for ISO/IEC WD TS 25025
         str(" ").maybe >> ((stage.as(:stage) | type) >> str(" ")).maybe >>
-        digits.as(:number) >> part.maybe >> iteration.maybe >>
+        digits.as(:number) >>
+        # for identifiers like ISO 5537/IDF 26
+        (str("/IDF") >> str(" ") >> digits).maybe >>
```

Comment:
This is "ISO 5537 | IDF 26", split with a pipe symbol, not a slash. This is what is called a "dual published" document (i.e. has 2 document identifiers) vs a "co-published" (i.e. 1 "joint document identifier").